index.html

```diff
@@ -18,19 +18,19 @@
                     <li class="nav-bar-li"><a href="#recent-sec" class="nav-a">Portfolio</a></li>
                     <li class="nav-bar-li"><a href="#about-sec" class="nav-a">About</a></li>
                     <li class="nav-bar-li"><a href="#contact-sec" class="nav-a">Contact</a></li>
-                    <li class="nav-bar-li"><a href="#contact-sec" class="nav-a"><img src="./images/mail_icon.png" alt=""></a></li>
+                    <li class="nav-bar-li"><a href="#contact-sec" class="nav-a"><img src="./images/mail_icon.png" alt="Mail icon"></a></li>
                     <li id="mobile-icon"><a href="#"><img src="./images/Union.png" alt=""></a></li>
                 </ul>
             </nav>
         </header>
         <div class="main-body">
             <div class="social-icon">
                 <ul>
-                    <li><a href="https://github.com/byiringiroscar"><img src="./images/github.png"></a></li>
-                    <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin.png"></a></li>
+                    <li><a href="https://github.com/byiringiroscar"><img src="./images/github.png" alt="github"></a></li>
+                    <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin.png" alt="linkedin"></a></li>
                     <li><a href="#"><img src="./images/hand.png"></a></li>
```

Comment:
- The image on line 31 is missing an `alt` text, please fix this so it helps to improve the accessibility of your site 🙏